lib/sql_parser.js

```diff
@@ -70,19 +70,22 @@ function getSelect(query) {
 }
 
 function getFrom(query) {
-  const from = query.match(/\sfrom\s+([\w([\])"'`.,/=<>\s]+$)/i)[1].replace(cutOff, '');
-  const name = from.match(/\s+as\s+(\w+)/i);
-  const path = from.slice(0, name.index).match(/(\w+)\/(\w+)/i);
+  const from = query.match(/\sfrom\s+([\w([\])\"'`.,/=<>\s]+$)/i)[1].replace(cutOff, '');
+  const name = getParamName(from);
+  const path = parseQueryPartial(from, name);
```

Comment:
@pj-infest please move definition of functions before usage, or use `function declaration`